src/helpers/figures/charts/chart_common.ts

```diff
@@ -503,25 +502,21 @@ export function getFullTrendingLineDataSet(
   defaultBorderColor.a = 1;
 
   const borderColor = config.color || lightenColor(rgbaToHex(defaultBorderColor), 0.5);
-  const backgroundRGBA = colorToRGBA(borderColor);
-  // @ts-expect-error
-  if (dataset?.fill) {
-    backgroundRGBA.a = LINE_FILL_TRANSPARENCY; // to support area charts
-  }
 
   return {
-    ...dataset,
```

Comment:
Was needed to keep some properties of the "parent" data series, like the vertical axis position (maybe it's the only one though, but this one is really needed at least)